src/Symfony/Component/Stopwatch/StopwatchEvent.php

```diff
@@ -177,7 +177,7 @@ public function getDuration()
 
         for ($i = 0; $i < $left; ++$i) {
             $index = $stopped + $i;
-            $periods[] = new StopwatchPeriod($this->started[$index], $this->getNow());
+            $periods[] = new StopwatchPeriod($this->started[$index], $this->getNow(), true);
```

Comment:
adding param to method is BC breaker.
you are thinking more about StopWatchPreciseEvent